src/matcha_ml/templates/azure_template/run_azure_template.py

```diff
@@ -272,6 +269,18 @@ def is_approved(self, verb: str) -> bool:
         print_status(summary_message)
         return typer.confirm(f"Are you happy for '{verb}' to run?")
 
+    def _write_outputs_state_cloud_only(self) -> None:
+        """Write the outputs of the Terraform deployment to the state JSON file."""
+        with open(self.state_file) as f:
+            state_file_data = json.load(f)
+
+        updated_state_file_data = {}
+        for key, value in state_file_data.items():
+            if key in ["cloud", "id"]:
```

Comment:
I guess we want to keep values related to `cloud_azure_prefix`, `cloud_azure_location` and `cloud_azure_resource_group` here using `cloud`. What does `id` look for?

Reply:
This means we keep the UUID for the `matcha.state` file